src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/PostReach.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE [dbo].[PostReach]
+(
+	[PostReachId] BIGINT NOT NULL CONSTRAINT PK_PostReach PRIMARY KEY IDENTITY, 
```

Comment:
Please Remove the blank space to make all items be aligned
Please add the column
PostId (BIG INT, NOT NULL, Foreign Key to Post)